prow/oss/cluster/monitoring/prow_podmonitors.yaml

```diff
@@ -151,3 +151,42 @@ spec:
   selector:
     matchLabels:
       app: crier
+---
+apiVersion: monitoring.gke.io/v1alpha1
+kind: PodMonitor
+metadata:
+  labels:
+    app: sub
+  name: sub
+  namespace: prow-monitoring
+spec:
+  podMetricsEndpoints:
+  - interval: 30s
+    port: metrics
+    scheme: http
+  namespaceSelector:
+    matchNames:
+    - default
+  selector:
+    matchLabels:
+      app: sub
+---
+apiVersion: monitoring.gke.io/v1alpha1
+kind: PodMonitor
+metadata:
+  labels:
+    app.kubernetes.io/name: kubernetes-external-secrets
+    app: kubernetes-external-secrets
+  name: kubernetes-external-secrets
+  namespace: prow-monitoring
+spec:
+  podMetricsEndpoints:
+  - interval: 30s
+    port: prometheus
```

Comment:
> Noticed that the metrics port was not added initially, which probably was the reason why podmonitor was not so happy last time

The pod monitor uses the `prometheus` port. I don't think 9090 is a valid port number for KES.